packages/platforms/delta/x86-64/x86-64-delta-ag8032/onlp/builds/src/module/src/fani.c

```diff
@@ -282,6 +287,7 @@ int onlp_fani_info_get(onlp_oid_t id, onlp_fan_info_t* info)
 		info->status |= ONLP_FAN_STATUS_PRESENT;
 		if (info->caps & ONLP_FAN_CAPS_GET_RPM) {
 			if (plat_os_file_read_int(&info->rpm, fan->rpm_get_path, NULL) < 0) {
+				AIM_LOG_ERROR("plat_os_file_read_int failed for ONLP_FAN_CAPS_GET_RPM rpm_path=%s", fan->rpm_get_path);
```

Comment:
there is no need the AIM_LOG_ERROR, otherwise it could output the errror log when the fan mode was not plugged in, and unplugged status of the psu is the normal status.

Reply:
switched to AIM_LOG_TRACE to give some insight on a higher log level (the following were treated the same way)